test/tasks/publish/test_local.py

```diff
@@ -29,7 +29,10 @@ def test_execute(self, exec_command):
         task = local.LocalStagingTask()
         task.execute(
             gapic_code_dir=os.path.expanduser('~/foo/bar'),
-            git_repo={'gapic_subpath': 'pubsub'},
+            git_repo={
+                'gapic_subpath': 'pubsub',
+                'location': 'api_client_staging.git',
```

Comment:
I don't think I fully understand the significance of underscores vs. hyphens in this situation. The git repo name is api-client-staging, so why do we need to support conversion from underscores?